ci/docker/linux_opensuse_tumbleweed

```diff
@@ -0,0 +1,19 @@
+FROM opensuse/tumbleweed
+RUN \
+  zypper -n update && \
+  zypper -n install \
+  shadow util-linux util-linux-systemd systemd-devel \
```

Comment:
Why do we need util-linux-systemd?

Reply:
Otherwise it fails when it run `lsblk` command.

Reply:
I think it'd be fine to just remove that...

Reply:
Done.